src/app/static/src/js/components/reportLog/runningReportDetails.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div>
+        <div v-if="reportLog">
+            <div id="report-log">
+                <div class="row pt-2">
+                    <div id="report-git-branch" v-if="reportLog.git_branch" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Git branch:</span>
+                            <span class="font-weight-bold">
+                            {{ reportLog.git_branch }}
+                        </span>
+                        </div>
+                    </div>
+                    <div id="report-git-commit" v-if="reportLog.git_commit" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Git commit:</span>
+                            <span class="font-weight-bold">
+                            {{ reportLog.git_commit }}
+                        </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="row pt-2">
+                    <div id="report-params" v-if="paramSize > 0" class="col-sm-auto">
+                        <span>Parameters:</span>
+                        <span>
+                        <div class="d-md-table-row row" v-for="(value, key) in reportLog.params">
+                            <span class="border border-secondary col-md">{{ key }}:</span>
```

Comment:
Please remove the colon - it's not in the mockup and having done some testing I think it clutters up the display
```suggestion
                            <span class="border border-secondary col-md">{{ key }}</span>
```

Reply:
Done